ros/build.sh

```diff
@@ -0,0 +1,4 @@
+cd ~/CarND-CapStone/ros
+catkin_make
+source devel/setup.bash
+roslaunch launch/styx.launch
```

Comment:
Here you are running a simulator. In this case, the script should be named differently, like `build_and_run_sim.sh` or `build_and_run_styx.sh`.

Reply:
True, we can have two separate scripts then